components/orderForm/OrderDetails.js

```diff
@@ -109,7 +117,9 @@ export default function OrderDetails({children}) {
   return (
     <>
       <h2 className="h-10 text-lg mb-2 block tracking-wide text-gray-600 font-bold">Order Details</h2>
-
+      <div className="flex flex-row mb-3 space-x-5"> 
+        {inventoryInfo?.map(infoStr => <ReactMarkdown className="mb-4 text-zinc-900 flex-grow border p-2 border-black rounded" components={smallMarkdownStyle} children={infoStr}></ReactMarkdown>)}
```

Comment:
```suggestion
        {inventoryInfo.map(infoStr => <ReactMarkdown className="mb-4 text-zinc-900 flex-grow border p-2 border-black rounded" components={smallMarkdownStyle} children={infoStr}></ReactMarkdown>)}
```
ok actually after testing - the `?.` doesn't work, so instead let's just set inventoryInfo to an empty array (`inventoryInfo = []`) if the if condition on line 23 is false